File: Backend_Ecommerce/src/main/java/com/commerce/datamodel/UserPayment.java
package com.commerce.datamodel;


import com.amazonaws.thirdparty.jackson.annotation.JsonIgnore;
import javax.persistence.Access;
import javax.persistence.AccessType;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.OneToOne;
import javax.persistence.Table;
import java.util.Date;
import lombok.Getter;
import lombok.Setter;

@Entity
@Table(schema = "public", name = "userpayment")
@Getter
@Setter
@Access(AccessType.FIELD)
public class UserPayment
{
  
  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  @Column(name = "id_payment")
  private int id;
  
  @Column(name = "payment_type")
  private String payment_type;
  
  @Column(name = "provider")
  private String provider;
  
  @Column(name = "accountno")
  private int accountno;
  
  @Column(name = "expiry")
  private Date expiry;
  
  @ManyToOne
  @JoinColumn(name = "id_user", referencedColumnName = "id_user")
  private User user;
  
}

File: C# Backend/E-commerce/E-commerce/Models/UserPayment.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace E_commerce.DataModel
{
    [Table("userpayment", Schema = "public")]
    public class UserPayment
    {
        [Key]
        [Column("id_payment")]
        public int Id { get; set; }

        [Column("payment_type")]
        public string PaymentType { get; set; }

        [Column("provider")]
        public string Provider { get; set; }

        [Column("accountno")]
        public int AccountNumber { get; set; }

        [Column("expiry")]
        public DateTime Expiry { get; set; }

        [ForeignKey("user")]
        [Column("id_user")]
        public int UserId { get; set; }
    }
}
